apps/fluent-tester/src/FluentTester/FluentTester.tsx

```diff
@@ -73,18 +71,28 @@ export const FluentTester: React.FunctionComponent<FluentTesterProps> = (props:
 
   const TestComponent = selectedTestIndex == -1 ? EmptyComponent : sortedTestComponents[selectedTestIndex].component;
 
+  const HeaderSeparator = Separator.customize((t) => ({
+    color: t.colors.bodyFrameDivider,
+    separatorWidth: 2,
```

Comment:
width 2? thats suprising.

Reply:
because natively, we do "half pixel"

Reply:
Native on which Platform? This was originally 1, but I can change it back